plaid/src/apis/npm/npm_web_client.rs

```diff
@@ -0,0 +1,827 @@
+
+        info!(
+            "Removing user [{}] from team [{}] on behalf of [{module}]",
+            params.user,
+            params.team.to_string()
+        );
+
+        let body = format!(
+            r#"{{"csrftoken": "{}"}}"#,
+            self.get_csrftoken_from_cookies()
+                .map_err(|_| ApiError::NpmError(NpmError::WrongClientStatus))?
+        );
+        self.client
+            .post(format!(
+                "{}/settings/{}/teams/team/{}/users/{}/delete",
+                NPMJS_COM_URL,
+                self.config.npm_scope,
+                params.team.to_string(),
+                params.user
+            ))
+            .header("Content-Type", "text/plain;charset=UTF-8")
+            .body(body)
+            .send()
+            .await
+            .map(|_| Ok(0))
+            .map_err(|_| ApiError::NpmError(NpmError::FailedToRemoveUserFromTeam))?
+    }
+
+    /// Remove a user from the npm organization
+    pub async fn remove_user_from_organization(
+        &self,
+        user: &str,
+        module: &str,
+    ) -> Result<i32, ApiError> {
+        #[derive(Serialize)]
+        struct RemoveUserPayload<'a> {
+            csrftoken: &'a str,
+        }
+
+        info!("Removing user [{user}] from npm organization on behalf of [{module}]");
+
+        self.login()
+            .await
+            .map_err(|_| ApiError::NpmError(NpmError::LoginFlowError))?;
+        let csrf_token = self
+            .get_csrftoken_from_cookies()
+            .map_err(|_| ApiError::NpmError(NpmError::WrongClientStatus))?;
+        let payload = RemoveUserPayload {
+            csrftoken: &csrf_token,
+        };
+
+        self.client
+            .post(format!(
+                "{}/settings/{}/members/{}/delete",
+                NPMJS_COM_URL, self.config.npm_scope, user
+            ))
+            .json(&payload)
+            .send()
+            .await
+            .map(|_| Ok(0))
+            .map_err(|_| ApiError::NpmError(NpmError::FailedToRemoveUserFromOrg))?
+    }
+
+    /// Invite a user to the npm organization.
+    ///
+    /// If `team` is specified then the user is added to that team upon accepting the invite.
+    /// If `team` is `None`, then the user is added to the default "developers" team.
+    pub async fn invite_user_to_organization(
+        &self,
+        params: &str,
+        module: &str,
+    ) -> Result<i32, ApiError> {
+        self.login()
+            .await
+            .map_err(|_| ApiError::NpmError(NpmError::LoginFlowError))?;
+        let params: InviteUserToOrganizationParams =
+            serde_json::from_str(params).map_err(|_| ApiError::BadRequest)?;
+        let team = params.team.unwrap_or(plaid_stl::npm::NpmTeam::Developers);
+
+        info!(
+            "Inviting user [{}] to npm organization and team [{}] on behalf of [{module}]",
+            params.user,
+            team.to_string()
+        );
+
+        let body = format!(
```

Comment:
This will allow injection. You should create a struct and serialize it from there

Reply:
Right. Sanitized user name and team name

Reply:
Done!